packages/timeline-state-resolver/src/integrations/httpSend/index.ts

```diff
@@ -366,10 +246,18 @@ export class HTTPSendDevice extends DeviceWithState<HTTPSendState, DeviceOptions
 					'EAI_AGAIN',
 				]
 
-				if (retryCodes.includes(err.code) && this._resendTime) {
-					const timeLeft = Math.max(this._resendTime - (Date.now() - t), 0)
-					await new Promise<void>((resolve) => setTimeout(() => resolve(), timeLeft))
-					this._defaultCommandReceiver(_time, cmd, context, timelineObjId, layer).catch(() => null) // errors will be emitted
+				if (retryCodes.includes(err.code) && this.options?.resendTime) {
+					const timeLeft = Math.max(this.options.resendTime - (Date.now() - t), 0)
+					setTimeout(() => {
+						this.sendCommand({
```

Comment:
Because this is run inside a `setTimeout`, `sendCommand` will return before the retry attempt is performed. Is this desired?
This does look to match what was done previously